content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,61 @@
+==== Configuration Form Modernization
+
+The transition from HTML `table` layout to HTML `div` layout in configuration forms requires changes in some plugins for compatibility.
+Jenkins administrators should update the installed plugins when they upgrade to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are updated to the most recent release.
+
+Popular plugins that need to be updated after upgrading to Jenkins 2.277.1 include:
```

Comment:
Not helpful without popular plugins not fixed yet, e.g. Authorize Project

Reply:
Removed in 1d0d15b2d8e5e1fb39efe9eea2da410f257d552d